### DISCUSSION TITLE
Couple of issues I ran into 2.5.59.0

### DISCUSSION THREAD

**Discussion by Newcal** [supermerill/SuperSlicer#3132] [Community contribution]
Gday,

I have some weird artifacts with supports. It creates supports where it is not needed. Got few issues with that.

<img width="379" alt="Screen Shot 2022-08-26 at 09 37 01" src="https://user-images.githubusercontent.com/18645222/186798056-ef88e72f-713d-4656-9066-8d74f4422df6.png">

Also, after a purge line, the nozzle will push more plastic than needed. It creates a huge blob of plastic on the bed.

I am using SS on 2018 MacBook Pro (None M1).

Cheers

**Comment by neophyl** [comment]
Instead of a discussion can you please open actual issues for the problems you have found.

**Reply by Newcal** [reply]
Hello, I am not used to GH. I'll have a look at it. Cheers
